source/matrix.c

```diff
@@ -121,7 +117,13 @@ static inline void pwmNextColumn() {
   rowsEnabled = 0;
   for (size_t keyRow = 0; keyRow < NUM_ROW; keyRow++) {
     const uint8_t ledIndex = ROWCOL2IDX(keyRow, currentColumn);
-    const led_t cl = ledColors[ledIndex];
+    led_t cl;
+    /* TODO: Maybe... weight with alpha? */
```

Comment:
I was considering additional alpha values to implement blinking (but went with a thread) or to solve priorities with foreground vs caps lock. But currently only zero, non-zero is distinguished, and there're two layers.